importer_client/python/tools/timesketch_importer.py

```diff
@@ -143,6 +143,12 @@ def upload_file(
         timeline = streamer.timeline
         task_id = streamer.celery_task_id
 
+        analyzers_name = config_dict.get("analyzers_name")
```

Comment:
This is always `None`. The config_dict setting below says the variable name is `analyzer_names` not `analyzers_name`.